File: AtCoder/AtCoder Beginner Contest/088/B/C/ABC_088_B_01.c
#include <stdio.h>
#include <stdlib.h>

int compareInt(const void* a, const void* b)
{
    const int* p_a = a;
    const int* p_b = b;

    return *p_b - *p_a;
}

int main(void)
{
    int alice = 0, bob = 0;
    int N;
    scanf("%d", &N);
    int a[N];
    for (int i = 0; i < N; i++) {
        scanf("%d", &a[i]);
    }

    qsort(a, sizeof(a)/sizeof(a[0]), sizeof(int), compareInt);

    for (int i = 0; i < N; i++) {
        alice += i % 2 == 0 ? a[i] : 0;
        bob   += i % 2 == 1 ? a[i] : 0;
    }
    printf("%d\n", alice - bob);

    return 0;
}

File: AtCoder/AtCoder Beginner Contest/088/B/Go/ABC_088_B_01.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	alice := 0
	bob := 0
	var N int
	fmt.Scanf("%d", &N)
	a := make([]int, N)
	for i := 0; i < N; i++ {
		fmt.Scanf("%d", &a[i])
	}

	sort.Slice(a, func(i, j int) bool { return a[j] < a[i] })

	for i, n := range a {
		if i % 2 == 0 {
			alice += n
		} else {
			bob += n
		}
	}

	fmt.Printf("%d\n", alice - bob)
}